integration/tests/cook/test_basic.py

```diff
@@ -1632,6 +1632,7 @@ def query():
 
     def test_retrieve_jobs_with_deprecated_api(self):
         pools, _ = util.pools(self.cook_url)
```

Comment:
How about naming them to `all_pools` and `active_pools`?

Reply:
Done